docs/usage/shoot_maintenance.md

```diff
@@ -0,0 +1,80 @@
+Especially for large clusters it may take several hours until a graceful rolling update of the worker nodes succeeds (also depending on the workload and the configured pod disruption budgets/termination grace periods).
+
+Internally, Gardener is subtracting `15m` from the end of the time window to (best-effort) try to finish the maintenance until the end is reached, however, it might not work in all cases.  
+
+If you don't specify a time window then Gardener will randomly compute it.
+You can change it later, of course.
+
+## Automatic Version Updates
+
+The `.spec.maintenance.autoUpdate` field in the shoot specification allows you to control how/whether automatic updates of Kubernetes patch and machine image versions are performed.
+During the maintenance, Gardener checks if there are newer versions available, and if you opt-in, then your shoot will be updated accordingly.
+
+```yaml
+spec:
+  maintenance:
+    autoUpdate:
+      kubernetesVersion: true
+      machineImageVersion: true
+```
+
+Please refer to [this document](../proposals/05-versioning-policy.md) that describes the exact versioning policy.
+
+## Cluster Reconciliation
+
+Gardener administrators/operators can configure the Gardenlet in a way that it only reconciles shoot clusters during their maintenance time windows.
+This behaviour is not controllable by end-users but might make sense for large Gardener installations.
+Concretely, your shoot will be reconciled regularly during its maintenance time window.
+Outside of the maintenance time window it will only reconcile if you change the specification or if you explicitly trigger it, see also [this document](shoot_operations.md).
+
+## Confine Specification Changes/Updates Roll Out
+
+Via the `.spec.maintenance.confineSpecUpdateRollout` field you can control whether you want to make Gardener roll out changes/updates to your shoot specification only during the maintenance time window.
+It is `false` by default, i.e., any change to your shoot specification triggers a reconciliation (even outside of the maintenance time window).
+This is helpful if you want to update your shoot but don't want the changes to be applied immediately. One example use-case would be a Kubernetes version upgrade that you want to roll out during the maintenance time window.
+Any update to the specification will not increase the `.metadata.generation` of the `Shoot` which is something you should be aware of.
+Also, even if Gardener administrators/operators have not enabled the "reconciliation in maintenance time window only" configuration (as mentioned above) then your shoot will only reconcile in the maintenance time window.
+The reason is that Gardener cannot differentiate between create/update/reconcile operations.
+
+⚠️  If `confineSpecUpdateRollout=true`, please note that if you change the maintenance time window itself then it will only be considered in the next but one maintenance.
```

Comment:
```suggestion
⚠️  If `confineSpecUpdateRollout=true`, please note that if you change the maintenance time window itself then it will only be effective after the upcoming maintenance.
```